package.json

```diff
@@ -69,6 +70,7 @@
     "eslint-plugin-react": "^6.10.2",
     "eslint": "^3.14.0",
     "extract-text-webpack-plugin": "~2.1.0",
+    "fs-extra": "^3.0.1",
```

Comment:
Well, it's a dev dependency, nobody dies. But if you really want to be cool, just do:

```bash
mkdir dist
touch dist/.gitkeep
```

And add to the .gitignore to:

```
!dist/.gitkeep
```